### PR TITLE
[New] Add stacktable.js@1.0.2 w/ git auto-update

### LINKED ISSUE #9486: [Request] Add stacktable
**Library name:** stacktable
**Git repository url:** https://github.com/johnpolacek/stacktable.js/
**License(s):** Dual MIT & GPL license
**Official homepage:** http://johnpolacek.github.io/stacktable.js/

### LINKED ISSUE #9589: [Request] Add Stackable.js
**Library name:** Stackable.js
**Git repository url:** https://github.com/johnpolacek/stacktable.js/
**npm package url(optional):** 
**License(s):** MIT+GPL
**Official homepage:** https://johnpolacek.github.io/stacktable.js/

### PR DESCRIPTION
**Pull request by sufuf3**
close #9486, close #9589, close #9711

Pull request for issue: #9486, #9589
Related issue(s): #9711 #11170

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/johnpolacek/stacktable.js
 * Official website (optional, not the repository): http://johnpolacek.github.io/stacktable.js/
 * NPM package url (optional): https://www.npmjs.com/package/stacktable.js
 * License and its reference: [MIT](https://github.com/johnpolacek/stacktable.js/blob/master/License.txt#L1)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 48
   - Count of stars: 897
   - Count of forks: 128
 * NPM download stats (optional):
   - Downloads in the last day: 0
   - Downloads in the last week: 18
   - Downloads in the last month: 129

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sufuf3** [comment]
@dakshshah96 @cdnjs/intern3
Please help me review this PR, Thank you.

**Comment by PeterDaveHello** [comment]
@sufuf3 btw, I think you can request review from teams now, on the reviewer list!

**Comment by PeterDaveHello** [comment]
@maruilian11 please take a look.

**Comment by maruilian11** [comment]
@sufuf3 Why don't you close the duplicated issue?

**Comment by maruilian11** [comment]
And It does has npm package: https://www.npmjs.com/package/stacktable.js

**Comment by sufuf3** [comment]
@maruilian11 
I found #9589 after I finished the commit, so I just add the issue number in the commit message.
Now, I add the Duplicate tag and leave the message https://github.com/cdnjs/cdnjs/issues/9589#issuecomment-312504724.
And I filled in the npm info in the first comment.

Please help me review this PR, Thank you.

File: ajax/libs/stacktable.js/package.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    "jquery",
+    "tables"
+  ],
+  "author": {
+    "name": "John Polacek",
+    "url": "http://www.johnpolacek.com"
+  },
+  "licenses": [
+    "MIT",
+    "GPL-3.0"
```
Comment (maruilian11) [inline ajax/libs/stacktable.js/package.json]: https://github.com/johnpolacek/stacktable.js/blob/master/License.txt#L24
Please confirm with the author if it's under `GPL-3.0` or `LGPL-3.0`.

Comment (sufuf3) [inline ajax/libs/stacktable.js/package.json]: It's under `LGPL-3.0` my bad. 😞

**Comment by sufuf3** [comment]
@maruilian11 I update the licenses field from `GPL-3.0` to `LGPL-3.0`, 
Please help me review this PR, Thank you.

**Comment by PeterDaveHello** [comment]
Where is your confirmation?
https://github.com/cdnjs/cdnjs/pull/11472#discussion_r125188245

I thought its author may want GPL not LGPL? https://github.com/johnpolacek/stacktable.js/issues/48

**Comment by sufuf3** [comment]
As the author said https://github.com/johnpolacek/stacktable.js/issues/61#issuecomment-313882499 this library is updated to under MIT license, only.
@dakshshah96 @cdnjs/intern3 
Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
@extend1994
I thought this PR is from old PR #9711 which match #11170 rule. So that's why I keep `#9711` in the commit message. (To note the reason that why I have to create another PR to deal with old PR and close it.)
But with the old PR close, as you said, maybe we don't need `#9711` in the commit message.
Now, I removed `#9711`, Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
@cdnjs/intern3 
Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
@dakshshah96  @cdnjs/intern3
Please help me review this PR, Thank you.

**Comment by sufuf3** [comment]
@pvnr0082t @maruilian11 
Please help me review this PR, Thank you.

**Pull request merged**